Update README in favor of renaming repo

ascendbruce opened the pull request:
https://github.com/taiwanese-work-in/site/issues/5
## Why is this change necessary?
- To better guide our potential contributors.
## How does it address the issue?
- Update README and .github templates.

ascendbruce committed:
Update README in favor of renaming repo

https://github.com/taiwanese-work-in/site/issues/5

* To better guide our potential contributors.

* Update README and .github templates.

ascendbruce committed:
Add code review guidelines

## Why is this change necessary?

- To clarify how to get code reviewed

## How does it address the issue?

- Update README

ascendbruce commented on README.md:
````diff
@@ -1,51 +1,40 @@
-- 
-```
+* 使用 [GitHub flow](https://guides.github.com/introduction/flow/)-改 (merge 後才 deploy production)。
+* Code Review 後覺得 ok 的話請給 thumbs up，直到有信心上 production 沒問題、code 可維護性沒問題時，由 author 自行 merge 即可。
````
嗯... 更精確來說是「由任一位 collaborator (可以是 author 本人) 進行 merge」?

ascendbruce replied:
> Code Review 後若無問題請給 `LGTM` 或 👍 ，視情況由任一位 collaborator (可以是 author 本人) 進行 merge。考慮情況如緊急度、修改的複雜度、feedback 狀況等。

ascendbruce commented:
應該可以再把 branching & PR policy 抽到 `.github/CONTRIBUTING.md`

yorkxin commented on README.md:
```diff
@@ -1,51 +1,40 @@
+* 使用 [GitHub flow](https://guides.github.com/introduction/flow/)-改 (merge 後才 deploy production)。
+* Code Review 後覺得 ok 的話請給 thumbs up，直到有信心上 production 沒問題、code 可維護性沒問題時，由 author 自行 merge 即可。
+* PR description 請使用 [此樣版](/.github/PULL_REQUEST_TEMPLATE.md)。
+* PR description 可以用中文寫，但 git commit message 請只使用英文。
```
reason?

ascendbruce replied:
hummm... 總覺得比較乾淨 (?)

ascendbruce replied:
考慮了一下，我支持用英文的點是因為 commit message 有一些主流的寫法 convention 例如在這篇裡指定了型態跟句型 http://chris.beams.io/posts/git-commit/ ，但用中文寫就比較難受到此限制

如果說 subject 必須用英文並 follow 該 convention，第三行起則可用中文，這樣如何？

yorkxin replied:
我的想法是因為這個 repo 主要是開給台灣人寫的，所以開放中文和英文會比較好。

至於 commit log 的規範，除非是大改程式，不然改錯字之類的不需要那麼嚴格，而且改程式的情況下，我們會讓我們信任的 developer 加入，這種情況下只要在 contribution 裡面稍微提醒一下就行了。我也不喜歡看到一大堆 WIP 😄

wayne5540 replied:
我是覺得不用太嚴格，靜態頁面要改很容易，所以要求開起來能看就好，不像程式邏輯要改很複雜所以要有漂亮的 commit #我的觀察啦

ascendbruce replied:
> PR description 與 git commit message 都可使用中文，但 commit message 的 subject line 建議使用英文並 follow http://chris.beams.io/posts/git-commit/ 慣例 (不強制)。

❔

yorkxin commented:
我們要不要先把 readme 的網址改掉之後再另外開一張票來寫 contribution.md？就像你說的 readme 裡面有些內容可以搬過去

ascendbruce commented:
可以，不過這張修改的內容我覺得還不行，今天晚上應該會再改一點

yorkxin commented:
好喔那先還給你

ascendbruce committed:
Refine contributing policy

## Why is this change necessary?

- According discussion
[here](https://github.com/taiwanese-work-in/foreign-country/pull/9#discu
ssion_r66815332) and
[here](https://github.com/taiwanese-work-in/foreign-country/pull/9#discu
ssion_r66945243)

## How does it address the issue?

- Change it

ascendbruce commented:
我想應該差不多了，若沒問題就來 merge?
